packages/website/src/components/ErrorsViewer.tsx

```diff
@@ -91,6 +91,18 @@ function ErrorBlock({
   );
 }
 
+function SuccessBlock(): JSX.Element {
+  return (
+    <div className={`admonition alert alert--success`}>
+      <div className="admonition-content">
+        <div className={clsx(styles.fixerContainer)}>
+          <div>There are no errors</div>
```

Comment:
I like the friendly phrasing on this option 😄 

```suggestion
          <div>All is ok!</div>
```